src/modules/popUp.js

```diff
@@ -29,7 +29,13 @@ const popUp = async (index) => {
   const p2 = document.createElement('p');
   p2.innerHTML = `<strong>Category:</strong> ${specificMeal.strCategory}`;
   div.appendChild(p2);
+  const divDown = document.createElement('a');
+  divDown.href = '#instructions';
+  divDown.className = 'scrollDown';
+  divDown.innerHTML = '<i class="fa-solid fa-circle-chevron-down"></i>';
+  div.appendChild(divDown);
   const p3 = document.createElement('p');
+  p3.id = 'instructions';
   p3.innerHTML = `<strong>Instructions:</strong> ${specificMeal.strInstructions}`;
```

Comment:
- [Optional ] Kindly try and shift the image above, so the arrow will show at first glance. good job so far 👏 

Your Design:
![yds](https://user-images.githubusercontent.com/50990925/196565160-2b3b4105-581f-4014-b9f6-8a79a40c2fb8.png)

Suggested Design:
![Suggested ](https://user-images.githubusercontent.com/50990925/196565213-b38cca58-ce28-4e63-a1ad-26150e677433.png)